fusor-ember-cli/app/models/deployment-plan.js

```diff
@@ -3,133 +3,22 @@ import DS from 'ember-data';
 
 export default DS.Model.extend({
   name: DS.attr('string'),
-  uuid: DS.attr('string'),
-  roles: DS.hasMany('deployment-role', {inverse: null, async: false}),
-  parameters: DS.hasMany('deployment-plan-parameter', {inverse: null, async: false}),
+  parameters: DS.attr(),
+  roles: DS.attr(),
```

Comment:
Ah yes, they don't actually have a valid roles anymore.  Egon has it [hardcoded](https://github.com/fusor/egon/blob/843667937cef53d3e64a736f352fb26613578e66/lib/egon/overcloud/undercloud_handle/deployment.rb#L138) to be:

```json
 "roles": [
            "Controller",
            "Compute",
            "BlockStorage",
            "ObjectStorage",
            "CephStorage"
        ]
```

As it's no longer an array of serialized objects, I had to use DS.attr.  Having said that, I ended up hardcoding roles and using them to store state on the assign-nodes controller, so I'm not really using the roles.